src/config.ts

```diff
@@ -106,14 +113,14 @@ export function defineSonarScannerParams(
   }
 
   // #3 - check what's passed in the call params - these are prevalent params
-  if (params.serverUrl) {
-    sonarScannerParams['sonar.host.url'] = params.serverUrl;
+  if (scanOptions.serverUrl) {
+    sonarScannerParams['sonar.host.url'] = scanOptions.serverUrl;
   }
-  if (params.token) {
-    sonarScannerParams['sonar.token'] = params.token;
+  if (scanOptions.token) {
+    sonarScannerParams['sonar.token'] = scanOptions.token;
   }
-  if (params.options) {
-    sonarScannerParams = Object.assign(sonarScannerParams, params.options);
+  if (scanOptions.options) {
+    sonarScannerParams = Object.assign(sonarScannerParams, scanOptions.options);
   }
```

Comment:
might have overlapped a bit on the params, so we can sync and make sure we get the version we want merged